src/codeu/chat/client/commandline/Chat.java

```diff
@@ -155,17 +176,17 @@ public void invoke(Scanner args) {
     //
     panel.register("u-sign-in", new Panel.Command() {
       @Override
-      public void invoke(Scanner args) {
-        final String name = args.hasNext() ? args.nextLine().trim() : "";
-        if (name.length() > 0) {
+      public void invoke(List<String> args) {
+        if(args.size() != 1){
+          System.out.println("ERROR: Failed to add new user");
```

Comment:
We are not adding new user in here. Here we are signing in. May be the error message something like "Missing usename" or something.